config/pipelines/pacbio.yml

```diff
@@ -423,6 +423,7 @@ default: &default
               for:
                 - :well
               with: :well
+      # v12
```

Comment:
assume this is going to be done in DPL-710

Reply:
yes, that's the sample sheet config story